tasks/main.yml

```diff
@@ -24,6 +24,7 @@
   shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
   with_items: "{{ plugins_to_install }}"
   register: plugin_response
+  when: elasticsearch_file_name|get_major_version < 5
```

Comment:
All of the plugins we use are site plugins though right? The only things mentioned in the docs are site plugins, or installing plugins [using kibana](https://www.elastic.co/guide/en/kibana/current/_installing_plugins.html)